frontend/Dockerfile

```
@@ -1,61 +1,28 @@
-# Stage 1: Builder
-# Utiliser une version LTS spécifique de Node.js (branche 22.x), variante slim recommandée
-FROM node:22.15.1-slim AS builder
+# Utiliser une version LTS spécifique de Node.js (branche 22.x), variante slim recommandée pour le développement
+FROM node:22.15.1-slim
 
 WORKDIR /app
 
 # Installer et configurer Corepack pour gérer PNPM
-# Mettre à jour Corepack vers la dernière version ou une version stable connue
 RUN npm install -g corepack@latest
 RUN corepack enable
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)